lib/utils/parse/parse.go

```diff
@@ -175,7 +175,12 @@ type Matcher interface {
 //
 // These expressions do not support variable interpolation (e.g.
 // `{{internal.logins}}`), like Expression does.
-func NewMatcher(value string) (Matcher, error) {
+func NewMatcher(value string) (m Matcher, err error) {
+	defer func() {
+		if err != nil {
+			err = trace.WrapWithMessage(err, "see supported syntax at https://gravitational.com/teleport/docs/enterprise/ssh-rbac/#node_labels-syntax")
```

Comment:
This URL doesn't go to a section - presumably this is the docs section you removed that @benarent is going to re-add into the 4.4 docs?

Reply:
nice catch, yeah this was the new section.
updated the link to point to the parent section, which is also appropriate (just less targeted).